packages/fontaine/src/index.ts

```
@@ -3,5 +3,7 @@ export { DEFAULT_CATEGORY_FALLBACKS, type FontCategory, resolveCategoryFallbacks
 export type { ResolveCategoryFallbacksOptions } from './fallbacks'
 export { getMetricsForFamily, readMetrics } from './metrics'
 
+export { transformCssFile } from './cli'
```

ESLint

[error] Expected "./cli" to come before "./metrics".

(perfectionist/sort-exports)